packages/fast-usdc/src/types.ts

```diff
@@ -55,4 +55,21 @@ export interface PoolMetrics extends PoolStats {
   shareWorth: Ratio;
 }
 
+export interface ChainPolicy {
+  nobleContractAddress: EvmHash;
+  cctpTokenMessengerAddress: EvmHash;
+  confirmations: number;
+  chainName: string;
```

Comment:
Should we also include `chainId: int;`? This seems like a more authoritative identifier than `chainName`, albeit less human-readable.

Edit: I see `chainId` is used as a key. Maybe it's better to make `chainName` they key?

Reply:
I just made chainName the key instead, since it's more self-describing. Having the chainId as the key might confuse people not knowing what the number means.